composer.json

```diff
@@ -3,74 +3,79 @@
-        "twig/extensions"                      : "^1.5",
-        "twig/twig"                            : "^1.28 || ^2.0",
-        "white-october/pagerfanta-bundle"      : "^1.0"
+        "php": "^7.1.3",
+        "ext-pdo_sqlite": "*",
+        "doctrine/doctrine-fixtures-bundle": "^2.3",
+        "erusev/parsedown": "^1.6",
+        "ezyang/htmlpurifier": "^4.9",
+        "sensio/framework-extra-bundle": "^3.0",
+        "symfony/asset": "^3.3",
+        "symfony/browser-kit": "^3.3",
+        "symfony/css-selector": "^3.3",
+        "symfony/expression-language": "^3.3",
+        "symfony/flex": "^1.0",
+        "symfony/form": "^3.3",
+        "symfony/framework-bundle": "^3.3",
+        "symfony/monolog-bundle": "^3.1",
+        "symfony/orm-pack": "^1.0",
+        "symfony/polyfill-apcu": "^1.4",
+        "symfony/security-bundle": "^3.3",
+        "symfony/swiftmailer-bundle": "^3.0",
+        "symfony/translation": "^3.3",
+        "symfony/validator": "^3.3",
+        "symfony/var-dumper": "^3.3",
+        "symfony/yaml": "^3.3",
+        "twig/extensions": "^1.5",
+        "twig/twig": "^2.4",
+        "white-october/pagerfanta-bundle": "^1.0"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle"            : "^3.0",
-        "friendsofphp/php-cs-fixer"            : "^2.0",
-        "sensio/generator-bundle"              : "^3.0",
-        "symfony/phpunit-bridge"               : "^3.0"
-    },
-    "scripts": {
-        "symfony-scripts": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
-        ],
-        "post-install-cmd": [
-            "@symfony-scripts"
-        ],
-        "post-update-cmd": [
-            "@symfony-scripts"
-        ]
+        "dama/doctrine-test-bundle": "^3.1",
+        "friendsofphp/php-cs-fixer": "^2.4",
+        "symfony/debug-bundle": "^3.3",
+        "symfony/dotenv": "^3.3",
+        "symfony/phpunit-bridge": "^3.3",
+        "symfony/profiler-pack": "^1.0",
+        "symfony/web-server-bundle": "^3.3"
     },
     "config": {
         "platform": {
-            "php": "5.5.9"
+            "php": "7.1.3"
+        },
+        "preferred-install": {
+            "*": "dist"
         },
         "sort-packages": true
     },
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\Tests\\": "tests/"
+        },
+        "files": [
+            "vendor/symfony/var-dumper/Resources/functions/dump.php"
```

Comment:
I guess this file can be removed here, because `var-dumper` is now required directly.